packages/cozy-harvest-lib/src/components/CozyConfirmDialogProvider/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { createContext, useContext, useState } from 'react'
+import Button from 'cozy-ui/transpiled/react/MuiCozyTheme/Buttons'
```

Comment:
Button is deprecated no?

Reply:
This is Buttons

Reply:
I was on my iphone for doing the first pass. It was hard 🤣